options/options.js

```diff
@@ -284,60 +283,47 @@ function loginGitLabOauth(param) {
       password: password
     }
   })
-    .done(response => {
-      return $.getJSON(
-        `${baseUrl}/user`,
-        { access_token: response.access_token }
-      )
-        .done(user => {
-          chrome.storage.sync.set({scm: param.scm, user: user.username, token: {type : 'oAuth', token :response.access_token}, baseUrl: baseUrl}, () => {
-            location.reload();
-          });
-          chrome.storage.local.get('tab', (item) => {
-            if(item.tab) {
-              chrome.tabs.reload(item.tab);
-            }
-          });
-        });
-    })
-    .fail(err => {
-      if (err.status == 401 &&
-        err.getResponseHeader('X-GitLab-OTP') !== null &&
-        $('.login-item-otp').filter(':visible').length == 0) {
-        $('.login-item').animate({height: 'toggle', opacity: 'toggle'}, 'slow');
-      } else {
-        $('.error').show();
-      }
-    })
+  .done(response => {
+    return $.getJSON(
+      `${baseUrl}/user`,
+      { access_token: response.access_token }
+    )
+    .done(user => {
+      chrome.storage.sync.set({scm: param.scm, user: user.username, token: {type : 'oAuth', token :response.access_token}, baseUrl: baseUrl}, () => {
+        location.reload();
+      });
+      chrome.storage.local.get('tab', (item) => {
+        if(item.tab) {
+          chrome.tabs.reload(item.tab);
+        }
+      });
+    });
+  })
+  .fail(err => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/c0b889de942411e3a53c4f3be8d2b855a8d0621a/options/options.js#L302"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>